pkg/operator/configobservation/etcd/observe_etcd.go

```diff
@@ -19,7 +19,7 @@ func ObserveStorageURLs(genericListers configobserver.Listers, currentConfig map
 	observedConfig = map[string]interface{}{}
 	storageConfigURLsPath := []string{"storageConfig", "urls"}
 	if currentEtcdURLs, found, _ := unstructured.NestedStringSlice(currentConfig, storageConfigURLsPath...); found {
-		unstructured.SetNestedStringSlice(observedConfig, currentEtcdURLs, storageConfigURLsPath...)
+		errs = append(errs, unstructured.SetNestedStringSlice(observedConfig, currentEtcdURLs, storageConfigURLsPath...))
```

Comment:
@deads2k was this missed intentionally?

Reply:
> @deads2k was this missed intentionally?

No, but check to make sure you are adding nils to the list.

Reply:
> No, but check to make sure you are adding nils to the list.

are not adding nils.  Still oustanding nil check needed.